### PR TITLE
Change endpoints to the ones that are used by airbyte UI

### PR DESCRIPTION
**Pull request by emilija-omnisend**

### CODE REVIEW

File: python_modules/libraries/dagster-airbyte/dagster_airbyte/resources.py [hunk 1]
```diff
@@ -171,10 +171,10 @@ def get_default_workspace(self) -> str:
         )
         return workspaces[0]["workspaceId"]
 
-    def get_source_definition_by_name(self, name: str, workspace_id: str) -> Optional[str]:
+    def get_source_definition_by_name(self, name: str) -> Optional[str]:
         name_lower = name.lower()
         definitions = self.make_request_cached(
-            endpoint="/source_definitions/list_for_workspace", data={"workspaceId": workspace_id}
+            endpoint="/source_definitions/list"
```
Comment (benpankow) [inline python_modules/libraries/dagster-airbyte/dagster_airbyte/resources.py:177]:
```suggestion
            endpoint="/source_definitions/list", data={}
```
data param is non-optional at the moment

File: python_modules/libraries/dagster-airbyte/dagster_airbyte/resources.py [hunk 2]
```diff
@@ -186,14 +186,13 @@ def get_source_definition_by_name(self, name: str, workspace_id: str) -> Optiona
...
-    def get_destination_definition_by_name(self, name: str, workspace_id: str):
+    def get_destination_definition_by_name(self, name: str):
         name_lower = name.lower()
         definitions = cast(
             Dict[str, List[Dict[str, str]]],
             check.not_none(
                 self.make_request_cached(
-                    endpoint="/destination_definitions/list_for_workspace",
-                    data={"workspaceId": workspace_id},
+                    endpoint="/destination_definitions/list"
```
Comment (benpankow) [inline python_modules/libraries/dagster-airbyte/dagster_airbyte/resources.py:195]:
```suggestion
                    endpoint="/destination_definitions/list", data={}
```
as above

**Review by benpankow — APPROVED** [review]
great!

File: python_modules/libraries/dagster-airbyte/dagster_airbyte/resources.py (hunk 1 above)
Comment (emilija-omnisend) [reply python_modules/libraries/dagster-airbyte/dagster_airbyte/resources.py]: fixed

File: python_modules/libraries/dagster-airbyte/dagster_airbyte/resources.py (hunk 2 above)
Comment (emilija-omnisend) [reply python_modules/libraries/dagster-airbyte/dagster_airbyte/resources.py]: fixed

**Pull request merged**